andela/ah-baqi-frontend
# #166980214 implement pagination

### What does this PR do? 
   - Adds infinite scroll pagination to view all articles.

### Description of Task to be completed?
- Checkout to:
      ft-pagination-166980214

- Install dependencies:

      yarn
     
- Run the local server:

      yarn develop
     
### How should this be manually tested?
- Open the local server URL on your browser and you will see a display of articles
- Scroll down and more articles will continue to unveil

### What are the relevant pivotal tracker stories?
[Pagination](https://www.pivotaltracker.com/story/show/166980214)
### Screenshots
<img width="1498" alt="Screenshot 2019-07-24 at 16 43 14" src="https://user-images.githubusercontent.com/45054246/61799118-2b07b000-ae33-11e9-94ab-091b4f13aec4.png">
<img width="842" alt="Screenshot 2019-07-24 at 16 43 32" src="https://user-images.githubusercontent.com/45054246/61799132-30fd9100-ae33-11e9-8e20-d712fdea355e.png">
<img width="1674" alt="Screenshot 2019-07-24 at 16 43 43" src="https://user-images.githubusercontent.com/45054246/61799135-30fd9100-ae33-11e9-9306-fb9053cc0d59.png">
